src/swap/letsexchange.js

```diff
@@ -0,0 +1,296 @@
+// @flow
+
+import {
+  asBoolean,
+  asEither,
+  asNumber,
+  asObject,
+  asOptional,
+  asString
+} from 'cleaners'
+import { lt } from 'biggystring'
+import {
+  type EdgeCorePluginOptions,
+  type EdgeCurrencyWallet,
+  type EdgeSpendInfo,
+  type EdgeSwapInfo,
+  type EdgeSwapPlugin,
+  type EdgeSwapQuote,
+  type EdgeSwapRequest,
+  type EdgeTransaction,
+  SwapBelowLimitError,
+  SwapCurrencyError
+} from 'edge-core-js/types'
+
+import {
+  type InvalidCurrencyCodes,
+  checkInvalidCodes,
+  makeSwapPluginQuote, getCodes,
+  getCodesWithMainnetTranscription
+} from '../swap-helpers.js'
+
+const pluginId = 'letsexchange'
+const swapInfo: EdgeSwapInfo = {
+  pluginId,
+  displayName: 'LetsExchange',
+  supportEmail: 'support@letsexchange.io'
+}
+
+const orderUri = 'https://api.letsexchange.io/?exchangeId='
```

Comment:
This URL does not bring the user to a valid order status page.

Reply:
this url show exchange info page, like this
<img width="1662" alt="image" src="https://user-images.githubusercontent.com/80786374/158781321-e1a48df1-0edb-444a-a929-2e4ee2c3457d.png">

Reply:
That was not my experience in testing. The links go to an API doc page.